package.json

```diff
@@ -33,7 +33,7 @@
     "prepublishOnly": "npm ci && npm test"
   },
   "config": {
-    "eslint": "--ignore-path .gitignore --cache --format=codeframe --max-warnings=0 \"{bin,.github}/**/*.{js,md,html}\" \".*.js\" \"*.md\"",
```

Comment:
I've disabled ESLint for Markdown files temporarily, because I was seeing a bunch of errors for the README file:

```
error: Resolve error: typescript with invalid interface loaded as resolver (import/no-self-import) at README.md:46:1:
  44 |
  45 | ```js
> 46 | const { overrides } = require('@netlify/eslint-config-node/vanilla_js_config')
     | ^
  47 |
  48 | module.exports = {
  49 |   extends: '@netlify/eslint-config-node/vanilla_js_config',
```

I still haven't figured out how to fix them, so until then I've disabled linting for `.md`.

Reply:
Installing `eslint-import-resolver-typescript` fixed it for me: https://github.com/netlify/zip-it-and-ship-it/pull/675/commits/1bfdfebeef3f87281c7ea1c5274ce3a3ba3aa1f9

Reply:
implemented in 08b75ac

Reply:
Is that something we could install in this repo instead?

Reply:
installed in 08b75ac :)

Reply:
absolutely! I did that in 08b75ac :)